recipes/napari/meta.yaml

```
@@ -0,0 +1,78 @@
+{% set name = "napari" %}
+{% set version = "0.2.12" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 56d41f596587b0877d31936972b8c53eab5e070e39377e1b97270d71f211793c
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  entry_points:
+    - napari = napari.__main__:main
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+  run:
+    - python >=3.6
+    - numpy
+    - scipy
+    - scikit-image
+    - vispy
+    - dask
+    - zarr
+    - fsspec
+    - imageio
+    - ipykernel
+    - jupyter_client <6.0.0
+    - qtpy
+    - qtconsole
+    - IPython
+    - backcall
+    - pluggy
+    - PyOpenGl
+    - PySide2
+    - wrapt
+    - numpydoc
+
+test:
+  requires:
+    - pytest
+    - pytest-faulthandler
+    - pytest-qt
+    - xarray
+    - pandas
+    - pytest-ordering
+    - pytest-timeout
+    - zarr
+  commands:
+    - setx QT_API "pyside2"  # [win]
+    - export QT_API=pyside2  # [unix]
+    - pytest --pyargs napari
```

Let's go ahead and comment these for now. We can debug the segfault in the feedstock as it seems unrelated to Napari (as it is pure Python).

```suggestion
    #- pytest --pyargs napari
```